template/node-gci/Dockerfile

```diff
@@ -52,6 +52,7 @@ ENV YOUNG_GEN_SIZE_MB="256"
 ENV YOUNG_GEN_SIZE_B="256000000"
 ENV EXPRESS_PORT="8083"
 ENV PROXY_PORT="8082"
+ENV GCI="true"
```

Comment:
We talked about it and agreed this env var is not needed, right? This template should be completely agnostic regarding gci

Reply:
This template is. That is node-gci's dockerfile.

Reply:
Didn't understand. There should be no env var controlling gci any environment. We talked about it the day before yesterday.

One template should use the gci and the other, should not know what it is.